### DISCUSSION TITLE
Demo of fireplace video...

### DISCUSSION THREAD

**Discussion by davepl** [PlummersSoftwareLLC/NightDriverStrip#497] [General]
Just a quick demo of the videoserver2.py script sending a fireplace video from YouTube to the matrix!

https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/3698967/1ef13eb6-7ff2-443c-b2a6-399b047e924c

**Comment by mikejohnau** [comment]
https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/26905608/7a9561c6-9528-4f30-8d47-8c1dc84e5542

Rick Rollin' here in Australia.....
